README

```diff
@@ -37,6 +37,14 @@
     The script will check your environment for tooling prerequisites and issue a warning/error 
     if any prerequisites are not installed on the system and cannot be installed automatically.
 
+    Since `avocado-setup.py` is run with root privileges, you need to
+    update `/etc/libvirt/qemu.conf` file to instruct libvirt to spawn
+    qemu command with these permissions:
+
+        user = "root"
+        group = "root"
+
```

Comment:
@mopsfelder I feel that is a default config option, as it works fine in the hostos installed machines in our env.
checkout, `dynamic_ownership = 1`, this again default values holds good, if anyone changes to 0 would cause the permission denied issues for image.

Reply:
My understanding reading https://github.com/open-power-host-os/tests/blob/master/README#L20 is that it supports CentOS 7.3 as the host.

When I ran avocado-setup.py on a CentOS host, I hit this issue because /etc/libvirt/qemu.conf had `#dynamic_ownership = 1` (commented out).

We shall either
(1) update README to say that avocado-setup.py should only be executed on a HostOS host (and remove CentOS as a supported host); or
(2) add instructions in README to properly configure dynamic_ownership.

What do you think, @sathnaga ?

Reply:
Good point.  The original wording is a bit misleading since the intention was to indicate that the Host tests should be run on the current CentOS-based HostOS.  We should just change that to "The latest version of HostOS".